test/request_storage_test.exs

```diff
@@ -23,6 +23,19 @@ defmodule RequestStorageTest do
     {:ok, %{crawler: :test_spider}}
   end
 
+  test "Starting a worker twice doesn't break", context do
+    Crawly.RequestsStorage.start_worker(:test_spider, "crawl_id")
+
+    request = %Crawly.Request{
+      url: "http://example.com",
+      headers: [],
+      options: []
+    }
+
+    assert :ok = Crawly.RequestsStorage.store(context.crawler, request)
```

Comment:
can we abstract these assertions to a `worker_is_healthy()` helper or something like that? seems repetitive to copy-paste them